### ISSUE TITLE
About the training step

### ISSUE THREAD

**Issue by leequan** [anson0910/CNN_face_detection#40]
Dear anson:
I have ran you code before and now I'd like to train my own caffee model.
But I don't understand the training step. For example,I do not know whether  the six networks can train at the same time. If training alone, where to get the input of net_12_cal, net_24c ? What is the difference between the face_12c and face_12c2 folder model?   What are the results of  * _SRquantize _ *. Caffemodel  and * _quantize _ *. Caffemodel   preserveing?  After 400000 iterations, accuracy = 0.5, loss = 0.64 almost unchanged  by training 12-net alone, what should be the result of this result.
So I do hope you can answer my question if convenient. Thank you very much!

**Comment by anson0910** [comment]
So all the nets should be trained alone, but input data for training higher resolution nets are dependent on lower resolution ones.
Calibration nets are not dependent on each other, so you can train those first.

Negative images to feed into 24-net are the negative images classified as positive by 12-net, and negative images to feed into 48-net are the negative images classified as positive by the cascade of 12-net and 24-net.

Steps to train the nets can be found in the [README](https://github.com/anson0910/CNN_face_detection).

You can ignore the files with _quantize_ in them, those are just used for my implementation on hardware.

Training 12-net can fail several times without a good initialization, you can just try retraining a few times.
The accuracy should be around 0.9 - 0.95.
